firstvoices/backend/tests/test_tasks/test_import_job_tasks.py

```diff
@@ -231,6 +237,36 @@ def test_boolean_variations(self):
             in validation_error_row.errors
         )
 
+    def test_existing_related_entries(self):
```

Comment:
I think we also need tests for uuids that are not found in the db, or uuids of other models